recipes/nspr/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+cd nspr
+
+export ARCH_FLAG=""
+if [[ "${ARCH}" == 64 ]]
+then
+    export ARCH_FLAG="--enable-64bit"
+fi
```

Comment:
Do you need this? If not, would drop it and just hard code `--enable-64bit` as we don't have any 32-bit builds any more in conda-forge. If you need it for local builds, we can certainly keep it.

Reply:
I don't need that, I'm going to change